GCP/Docker/scripts/bootstrap.backend.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+set -e
+
+# Function to add any cleanup actions
+function cleanup() {
+    echo "Cleanup."
+}
+trap cleanup EXIT
+
+# Initialize gcloud
+
+# Get the secret from secret manager
+function get_secret() {
+    local secret_name="$1"  
+    : "${json_secret:=$(gcloud secrets versions access latest --secret="$secret_name" --format='get(payload.data)' | tr '_-' '/+' | base64 -d)}"
+    local value=$(echo "$json_secret")
+    echo "$value"    
+}
+
+# Get the value from the secret based on the key
+function get_secret_value() {
+    local secret="$1" key="$2" form="$3" 
+    if [ "$form" == "json" ]; then
+        : "${final_value:=$(echo $secret | jq -r ."$key")}"
+  		echo "$final_value"
+    else
+        echo "$value"
+    fi
+}
+
+
+
+echo "Fetching variables from Secret Manager..."
+
+WEB_SECRETS="$(get_secret "webapp-secrets")"
+# DEMO_SECRETS=$(get_secret "demo-account")
+SENDGRID_SECRETS=$(get_secret "sendgrid-secrets")
+AIRBYTE_SECRETS=$(get_secret "airbyte-secrets")
+DALGO_DB_CREDENTIALS=$(get_secret "dalgo-db-credentials")
+
+
+# Fetch secret from Secret Manager at runtime
```

Comment:
nit: thinking if there's an opportunity for us to put the secret names in a file and then use a loop here to fetch the values from the secrets manager to reduce the repeated code